### PR TITLE
feat: Add the javascript file export options

### PR DESCRIPTION
**Pull request by balogun14**

### CODE REVIEW

File: src/sourceFiles/index.ts [hunk 1]
```diff
@@ -697,11 +697,13 @@ export namespace SourceFilesModel {
...
     export interface GeneralExportOptions {
         exportPattern: string;
+        exportQuotes?: ExportQuotes;
     }
 
     export interface PropertyExportOptions {
         escapeQuotes: EscapeQuotes;
         exportPattern: string;
+        exportQuotes?: ExportQuotes;
     }
```
Comment (andrii-bodnar) [inline src/sourceFiles/index.ts:707]: These types shouldn't be modified. You need to create a separate one - `JavaScriptExportOptions`

File: src/sourceFiles/index.ts [hunk 2]
```diff
@@ -711,6 +713,10 @@ export namespace SourceFilesModel {
         THREE = 3,
     }
 
+    export enum ExportQuotes {
+        SINGLE = 'single',
+        DOUBLE = 'double',
+    }
```
Comment (andrii-bodnar) [inline src/sourceFiles/index.ts:719]:
```suggestion
    export enum ExportQuotes {
        SINGLE = 'single',
        DOUBLE = 'double',
    }

```

File: src/sourceFiles/index.ts (hunk 1 above)
Comment (balogun14) [reply src/sourceFiles/index.ts]: I will do that now

**Review by balogun14 — COMMENTED** [review]
I created a new interface for JavaScriptExportOptions

File: src/sourceFiles/index.ts [hunk 3]
```diff
@@ -703,6 +703,10 @@ export namespace SourceFilesModel {
         escapeQuotes: EscapeQuotes;
         exportPattern: string;
     }
+    export interface JavaScriptExportOptions {
+        exportPattern: string;
+        exportQuotes?: ExportQuotes;
+    }
```
Comment (andrii-bodnar) [inline src/sourceFiles/index.ts:709]:
```suggestion

    export interface JavaScriptExportOptions {
        exportPattern: string;
        exportQuotes?: ExportQuotes;
    }
```

**Review by andrii-bodnar — APPROVED** [review]
@balogun14 thank you!

**Pull request merged**